amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -1852,101 +1852,713 @@ VX_API_CALL vx_node VX_API_CALL vxExtrppNode_NopbatchPD(vx_graph graph, vx_image
     return node;
 }
 
-VX_API_CALL vx_node VX_API_CALL vxExtrppNode_SequenceRearrange(vx_graph graph, vx_tensor pSrc, vx_tensor pDst, vx_array newOrder, vx_scalar layout) {
+VX_API_CALL vx_node VX_API_CALL vxRppSequenceRearrange(vx_graph graph, vx_tensor pSrc, vx_tensor pDst, vx_array pNewOrder, vx_scalar layout) {
```

Comment:
Please check if it is better to include node in name like
vxRppNodeSequenceRearrange

Reply:
Rajy suggested to name without node as vxRppBrightness, hence Node is not added here